docs/designs/helmify.md

```diff
@@ -0,0 +1,44 @@
+# Helm-ify
+
+## The problem
+
+Currently the project relies on helm install-ing several helm Charts,
+required for the various components. 
+However, other parts of the project such as the project components themselves 
+and migrations are a combination of `kustomize.yaml` manifests (components), k8s
+ yamls applied through `draconctl` (migrations). This is significantly confusing
+and tech debt since it is not clear what installs what, how and why.
+
+It is suggested that the project uses Helm as its sole way of deploying things.
+
+## Changes suggested
+
+The following changes are suggested
+
+* Make a `dracon-dev` chart which installs all the sub-charts referenced by 
+the `make-dev` target
+* Make `draconctl migrations apply` only apply migrations against a database url
```

Comment:
This doesn't make sense. The `draconctl migrations apply` always connects to a database using a URI and applies a set of migrations. Whether that happens while on an admins machine or inside a pod is irrelevant. The whole point of this work to not have a separate manual stage for running the command that deploys itself as a pod but to have it as an automated part of the Helm package. And why should it not be a job deployed by Helm? Helm can monitor jobs and track their progress.